src/test/java/bio/terra/service/filedata/azure/tables/TableDirectoryDaoConnectedTest.java

```diff
@@ -106,6 +127,7 @@ public void testStorageTableMetadataDuringFileIngest() {
     boolean deleteEntry =
         tableDirectoryDao.deleteDirectoryEntry(tableServiceClient, fileEntry1.getFileId());
     assertThat("Delete Entry 1", deleteEntry, equalTo(true));
+    directoryEntriesToCleanup.remove(fileEntry1.getFileId());
```

Comment:
This seems a bit fragile, what if we didn't remove these entries and instead caught the exception around each call to `deleteDirectoryEntry()` in the cleanup? That way we wouldn't have to remember to remove entries as they get deleted by the test.